### PR TITLE
Minor fixes to Tests and investigation on #6

### LINKED ISSUE #6: consistency in returned types
It's not clear when we should take:
```
pytorch_model(torch.from_numpy(X))[0]  #  RandomForestClassifier
```
vs. say
```
pytorch_model(torch.from_numpy(X))[1]   # DecisionTreeClassifier
```

In the end, we want it to be something simple like what we compare to such as
```
model.predict(X)
```

### PR DESCRIPTION
**Pull request by interesaaat**
Fixes #6.

### CODE REVIEW

File: tests/test_lightgbm_converters.py [hunk 1]
```diff
@@ -1,7 +1,8 @@
 """
-Tests sklearn TruncatedSVD converter
+Tests Skleran LightGBM converters.
```
Comment (ksaur) [inline tests/test_lightgbm_converters.py]: fix the "skleran" -> "sklearn"

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 numpy>=1.15
 scikit-learn==0.21.3
-torch==1.3.1
+torch>=1.3.1
```
Comment (ksaur) [inline requirements.txt]: let's fix to 1.4.0 so that it works with python 3.8

File: tests/test_lightgbm_converters.py [hunk 3]
```diff
@@ -60,25 +75,21 @@ def _run_lgbm_regressor_converter(self, num_classes, extra_config={}):
...
-
-    # multi
-    def test_lgbm_multi_regressor_converter(self):
-        self._run_lgbm_regressor_converter(3)
+        self._run_lgbm_regressor_converter(1000)
 
-    # gemm
+    # gemm regressor
     def test_lgbm_gemm_regressor_converter(self):
-        self._run_lgbm_regressor_converter(3, extra_config={"tree_implementation": "gemm"})
```
Comment (ksaur) [inline tests/test_lightgbm_converters.py:73]: just curious, how much longer do these take to run?

File: tests/test_sklearn_random_forest_converters.py [hunk 4]
```diff
@@ -27,26 +27,38 @@ def _run_random_forest_classifier_converter(self, num_classes, extra_config={},
...
+    # perf_tree_trav classifier
+    def test_random_forest_perf_tree_trav_classifier_converter(self):
+        self._run_random_forest_classifier_converter(2, extra_config={"tree_implementation": "perf_tree_trav"})
+
     # multi classifier
     def test_random_forest_classifier_multi_converter(self):
         self._run_random_forest_classifier_converter(3)
 
-    # gemm classifier
+    # gemm multi classifier
```
Comment (ksaur) [inline tests/test_sklearn_random_forest_converters.py:53]: should we also make these 1000 to make the other file?

File: tests/test_lightgbm_converters.py (hunk 1 above)
Comment (interesaaat) [reply tests/test_lightgbm_converters.py]: fixed.

File: tests/test_lightgbm_converters.py (hunk 3 above)
Comment (interesaaat) [reply tests/test_lightgbm_converters.py:73]: The value here does not matter for the runtime. It's not like binary vs multiclass. In that case it matters :)

File: tests/test_sklearn_random_forest_converters.py (hunk 4 above)
Comment (interesaaat) [reply tests/test_sklearn_random_forest_converters.py:53]: No no this is ok. 1000 was to get wider margins to the regression.

**Pull request merged**